gemcalibration/src/common/CalibrationWeb.cc

```diff
@@ -0,0 +1,394 @@
+                                << "<label> <input type=\"checkbox\" class=\"check\" name=\""
+                                <<  t_stream.str() << "\" id=\"" << t_stream.str() 
+                                << "\">" << t_stream.str() << "</label>"
+                            << " </div>"
+                        << "</div>"
+                        << "<div class=\"col-md-9\">"
+                            << "<div class=\"dropdown\">"
+                                << "<button id=\"amc_dropdown_button\" class=\"btn btn-lg btn-outline dropdown-toggle\" data-toggle=\"dropdown\">Select AMC and OH</button>"
+                                << "<div class=\"dropdown-menu pre-scrollable\">" << std::endl;
+                                    for (unsigned int j = 0; j < NAMC; ++j) { //SHELF.AMC
+                                        t_stream.clear();
+                                        t_stream.str(std::string());
+                                        t_stream << "shelf"<< std::setfill('0') << std::setw(2) << i+1 << ".amc" << std::setfill('0') << std::setw(2) << j+1;
+                                        *out << "<span class =\"dropdown-item-text\">"
+                                            << "<div class=\"row\">"
+                                                << "<div class=\"col-md-3\">"
+                                                    << "<div class=\"checkbox\"><label><input type=\"checkbox\" class=\"check\" name=\""
+                                                    << t_stream.str() << "\" id=\"" << t_stream.str() << "\">  AMC"
+                                                    << std::setfill('0') << std::setw(2) << j+1 << " </label> </div>"
+                                                << "</div>"
+                                                << "<div class=\"col-md-1\">"
+                                                << "</div>"
+                                                << "<div class=\"col-md-8\">" << std::endl;
+                                                    t_stream << ".ohMask";
+                                                    *out << "<input type=\"text\" value=\"0x000\" size=\"4\" name=\""
+                                                    << t_stream.str() << "\" id =\"" << t_stream.str() << "\">OH mask </input>"
+                                                << "</div>"
+                                            << "</div>"
+                                        << "</span>" << std::endl;
+                                    } // end loop over NAMC 
+                                *out << "</div>"
+                            << "</div>" // end <div class="dropdown">
+                        << "</div>" // end <div class="col">
+                    << "</div>" << std::endl; // end <div class="row">
+                } // end loop over NSHELF
+            *out << "</div>" << std::endl; // end container
+        *out << "</form>"<< std::endl;
```

Comment:
This entire file should be reformatted, and subsequently, all trailing whitespace removed, before I can perform further review.